components/Form/EditSensorForm.jsx

```diff
@@ -0,0 +1,130 @@
+'use client'
+
+import { useState } from 'react'
+import { useRouter } from 'next/navigation'
+
+import { Input } from '@/components/ui/Input/Input'
+import { Button } from '@/components/ui/Button/Button'
+import { Select } from '@/components/ui/Select/Select'
+
+import { PageHeader } from '@/components/Page/PageHeader'
+import { Typography } from '@/components/ui/Typography'
+import { StationType } from '@prisma/client'
+import { useTranslations as getTranslations } from 'next-intl'
+
+export default function EditSensorForm({
+  id,
+  identifier,
+  type,
+  nbr_measures,
+  station_id,
+  records,
+  files,
+}) {
+  const [newIdentifier, setNewIdentifier] = useState(identifier)
+  const [newType, setNewType] = useState(type)
+  // const [newNbr_measures, setNewNbr_measures] = useState(nbr_measures)
+  // const [newStation_id, setNewStation_id] = useState(station_id)
+  // const [newRecords, setNewRecords] = useState(records)
+  // const [newFiles, setNewFiles] = useState(files)
+
+  const router = useRouter()
+
+  // const t = getTranslations('Sensor')
+
+  const handleSubmit = async (e) => {
```

Comment:
To manage the request with the api, you can use react-query as implemented into the [signup page](https://github.com/astrolabe-expeditions/data-platform/blob/dev/app/%5Blocale%5D/auth/signup/page.jsx) also